css-layout-classes/styles.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.row {
+  display: flex;
+}
+
+img {
+  width: 100%;
+}
+
+.col-3 {
+  width: 100%;
+}
+
+.col-one-third {
+  width: 100%;
+}
```

Comment:
Neither of these column classes have styling that matches their description. You need a rule for each column class you use, and styling that correlates to the name you gave it. Both of these rules have styling that would correlate with something like `column-full`, not `col-one-third` or `col-3`